test/resources/application.conf

```diff
@@ -34,6 +34,9 @@ limits {
   maxHouseConsignmentItem = 3
   maxPackages = 3
   maxHouseConsignments = 3
+  houseConsignment {
+      maxDepartureMeansOfTransport = 999
```

Comment:
Sorry, I should have been more clear. This should be a lower value in here. Something like 3 would be fine. Basically, it means if we ever run tests that generate using arbitrary house consignment departure transport means it will only create 3, instead of 999. Means the tests will complete quicker and less memory used.